app/routes/circulars/circulars.lib.ts

```diff
@@ -49,3 +53,46 @@ export const validSubjectKeywords = [
   'XRB',
   'XTR',
 ]
+
+export const emailAutoReplyChecklist = [
+  'this is an automatic reply',
+  'automatic reply: ',
+  'auto reply',
+  'autoreply',
+  'vacation',
+  'out of the office',
+  'out of office',
+  'out of town',
+  'away from my mail',
+  'away from his e-mail',
+  'away from her e-mail',
+  'away from the office',
+  'away from his office',
+  'away from her office',
+  'traveling until',
+  'no longer receiving mail',
+  'delivery failure notif',
+  'mail delivery failure',
+  'returned mail',
+  'saxzlcnkgzmfpbhvyzsbub',
+  'ponse_automatique',
+  'off-line re:',
+  're: ',
+  'fwd: ',
+  ' r: ',
+  ' ris: ',
+]
+
+export function formatAuthor({
+  name,
+  affiliation,
+  email,
+}: {
+  name?: string
+  affiliation?: string
+  email: string
+}) {
+  if (!name) return email
+  else if (!affiliation) return `${name} <${email}>`
+  else return `${name} at ${affiliation} <${email}>`
+}
```

Comment:
There are now two copies of this function. Please remove the other copy and update imports.